namespaces/mson-namespace.md

```diff
@@ -0,0 +1,503 @@
+# MSON Namespace
+
+This document extends [Refract][]'s [JSON Namespace][] with elements necessary to
+build [MSON][] DOM.
```

Comment:
Well I quite like it. Since it actually is DOM. Isn't ? Frankly over the years what I have learned is that AST is sort of alien to many, but everybody has a notion of what a DOM is. 

Is there any bad connotation to it? I don't know. 

I also like the "MSON refract" term you suggest at the bottom of this review, however, I would say I would much rather call these "namespaces" refracts (instead of namespaces)...